.github/workflows/test.yml

```diff
@@ -0,0 +1,62 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - master
+      - migrate-github-actions
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "12"
+      - run: sudo apt-get install xvfb
+        if: ${{ matrix.os == 'ubuntu-latest' }}
+      - run: yarn
+      - run: xvfb-run yarn ci
+        if: ${{ matrix.os == 'ubuntu-latest' }}
+      - run: yarn ci
+        if: ${{ matrix.os != 'ubuntu-latest' }}
+  integration-test:
+    runs-on: ubuntu-latest
```

Comment:
integration test needs MySQL & Postgres.
However GitHub actions disallow macos & windows to use services.

> Note: If your workflows use Docker container actions or service containers, then you must use a Linux runner:    If you are using GitHub-hosted runners, you must use an Ubuntu runner.
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idservices